casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -417,7 +417,8 @@ object BlockAPI {
           Seq[BlockHash],
           Float,
           Float,
-          Seq[Bond]
+          Seq[Bond],
+          Seq[ProcessedDeploy]
```

Comment:
Hmm, seems kind of wasteful to hold the complete `DeployData` when we just want the cost?

Reply:
Only the cost would be very unclear because there might be a lot of deploys in one block.

Kent suggested providing with [the deploydata](https://github.com/rchain/rchain/pull/2274#discussion_r266327615).